qa/tasks/qemu.py

```diff
@@ -203,14 +203,17 @@ def generate_iso(ctx, config):
                     'device_letter' not in disk or \
                     'image_url' in disk:
                 continue
-            dev_letter = disk['device_letter']
+            if disk['encryption_format'] == 'none':
```

Comment:
nit: probably add a comment here describing why we need to do this, so an occasional code reader would not have to look through the commit log to understand the reason. And additionally it would a reminder to change this in future.

Reply:
Ah, @ideepika already comment about a comment :-)